reporter/src/main/kotlin/Reporter.kt

```diff
@@ -22,16 +22,14 @@ package org.ossreviewtoolkit.reporter
 import java.io.File
 import java.util.ServiceLoader
 
-import org.ossreviewtoolkit.model.AnalyzerResult
 import org.ossreviewtoolkit.model.OrtResult
-import org.ossreviewtoolkit.model.ScanRecord
 import org.ossreviewtoolkit.model.config.PathExclude
 import org.ossreviewtoolkit.model.config.ScopeExclude
 import org.ossreviewtoolkit.utils.common.joinNonBlank
 
 /**
- * A reporter that creates a human-readable report from the [AnalyzerResult] and [ScanRecord] contained in an
- * [OrtResult]. The signatures of public functions in this class define the library API.
+ * A reporter that creates a human-readable report from a given [OrtResult]. The signatures of public functions in this
+ * class define the library API.
```

Comment:
If you are at it anyway you could also remove the second sentence because it just states the obvious.